jobs/config.json

```diff
@@ -5275,6 +5275,24 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7": {
+    "args": [
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7.env",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.6",
+      "--deployment=kubernetes-anywhere",
+      "--mode=local",
+      "--timeout=300m",
+      "--extract=ci/latest-1.6",
```

Comment:
So my question basically is, where is the branch for a job set?
The hard thing here is that kubeadm should be built from the v1.7 branch, but the e2e tests should be built from the v1.6 branch.

I trust that the thing you have here is correct, but I just want to understand the process better.